### PR TITLE
Improve compatibility 4 Animation

### PR DESCRIPTION
**Pull request by SadieYuCN**
1.use nine old android lib for Animation
2.update gradle version to 1.2.3

### CODE REVIEW

**Comment by carlonzo** [comment]
thanks

**Comment by SadieYuCN** [comment]
:)

**Comment by carlonzo** [comment]
I've changed my mind... why not!? I think I'll merge it. I've a PR ready to push then I will merge this

**Comment by medyo** [comment]
I implemented nineoldandrois locally then i discovered that is already done on this PR, I think @carlonzo should approve it since not all apps are using android 11 as min version.

**Comment by carlonzo** [comment]
I will!

File: build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.2.3'
+        classpath 'com.android.tools.build:gradle:1.3.0'
```
Comment (carlonzo) [inline build.gradle]: 1.3.1

File: core/src/main/java/it/carlom/stikkyheader/core/HeaderAnimator.java [hunk 2]
```diff
@@ -65,4 +66,13 @@ public int getHeightHeader() {
     public int getMaxTranslation() {
         return mMaxTranslation;
     }
+
+    public static void removeGlobalLayoutListener(ViewTreeObserver.OnGlobalLayoutListener victim,View view) {
```
Comment (carlonzo) [inline core/src/main/java/it/carlom/stikkyheader/core/HeaderAnimator.java]: please remove it. it does not fix the issue

**Comment by carlonzo** [comment]
I will apply a different solution. thanks for your PR!

**Comment by carlonzo** [comment]
@SadieYuCN here it is. let me know what you think

**Comment by SadieYuCN** [comment]
just like what @medyo said since not all apps are using android 11 as min version, so much better now :D

**Comment by carlonzo** [comment]
@SadieYuCN if you want to review https://github.com/carlonzo/StikkyHeader/pull/59 I have not fully tested it.

**Pull request closed without merging**